### ISSUE TITLE
Add Event Slots: OnSpaw, OnLoad

### ISSUE THREAD

**Issue by VectorWolf** [AgoutiGames/RawSalmonEngine#54]
On some occasions we need an event only once, upon construction of the object.
- For Actors simply add the OnSpawn event slot and initialize the event pipeline with this event.
- For the Map this is the first event slot at all, make sure to also add the OnAlways slot.

**Comment by VectorWolf** [comment]
Added the onload slot for the map at edc1dbaa1f9b4cd57121b76c322d9969aec290bf . OnAlways slot for map is low priority now.
